lasagne/layers/helper.py

```diff
@@ -334,10 +341,12 @@ def get_all_params(layer, **tags):
 
     Notes
     -----
-    If any layer's parameter was set to a Theano expression instead of a shared
-    variable, the shared variables involved in that expression will be returned
-    rather than the expression itself. Tag filtering considers all variables
-    within an expression to be tagged the same.
+    If any of the layers' parameters was set to a Theano expression instead
+    of a shared variable, `unwrap_shared` controls whether to return the
+    shared variables involved in that expression (``unwrap_shared=True``,
+    the default), or the expression itself (``unwrap_shared=False``). In
+    either case, tag filtering applies to the expressions, considering all
+    variables within an expression to be tagged the same.
     >>> import theano
```

Comment:
Just noticed that this example never rendered properly, but I'll fix it afterwards.